flink-filesystems/flink-s3-fs-presto/src/test/java/org/apache/flink/fs/s3presto/PrestoS3FileSystemITCase.java

```diff
@@ -49,6 +48,8 @@
 @RunWith(Parameterized.class)
 public class PrestoS3FileSystemITCase extends AbstractHadoopFileSystemITTest {
 
+    private static final String S3_USE_INSTANCE_CREDENTIALS = "presto.s3.use-instance-credentials";
```

Comment:
It seems that this constant has been moved and made private, so we need to declare it ourselves.